pages/windows/choco-list.md

```diff
@@ -0,0 +1,27 @@
+# choco list
+
+> Display a list of packages with Chocolatey.
+
+- Display all available packages:
+
+`choco list`
+
+- Display all locally installed packages:
+
+`choco list --local-only`
+
+- Display a list including local programs:
+
+`choco list --include-programs`
+
+- Display only approved packages:
```

Comment:
What does approved packages mean ?

Reply:
Approved packages are ones that have been marked as trusted.

For example, if they have been uploaded by the original vendor, or someone with a reputation for safe and high quality packages.

https://chocolatey.org/faq#what-is-a-trusted-package

Reply:
Ok